scripts/laser_csc.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import signal
+
+from lsst.ts.laser.statemachine import LaserCSC
+
+__all__ = ["main"]
+
+LOG_LEVEL = [logging.ERROR, logging.INFO, logging.DEBUG]
+
+
+def create_parser():
+    """Create parser
+    """
+    description = ["This is the main driver script for the LSST OCS scripts."]
+
+    parser = argparse.ArgumentParser(usage="run_sequence.py [options]",
+                                     description=" ".join(description),
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("-v", "--verbose", dest="verbose", action='count', default=0,
+                        help="Set the verbosity for the console logging.")
+    parser.add_argument("-c", "--console-format", dest="console_format", default=None,
+                        help="Override the console format.")
+    parser.add_argument("port",metavar="PORT",nargs=1,type=str,help="The port of the laser")
+
+    return parser
+
+
+def main(args):
+    """
+    Main method to startup OCS scripts in python.
+    :param args:
```

Comment:
If there are no program arguments there's no need to put these in the docstring.